libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -468,6 +468,7 @@ float AP_Baro::get_external_temperature(const uint8_t instance) const
     }
     
 #ifndef HAL_BUILD_AP_PERIPH
+#if AP_AIRSPEED_ENABLED
```

Comment:
I reckon at some stage someone will want to remove the condition AP_Periph thing here.  But doesn't need to be done as part of this PR as it is a behavioural change...